pyblish/plugin.py

```diff
@@ -1284,19 +1285,33 @@ def discover(type=None, regex=None, paths=None):
                 continue
 
             for plugin in plugins_from_module(module):
-                if plugin.__name__ in plugins:
+                # Check for duplicate plugin names. This is to preserve
+                # backwards compatility.
+                allow_duplicates = bool(
```

Comment:
This can be evaluated on module-import, no need to do it every iteration of this loop.

Reply:
The issue with putting it at the module level, is that the tests can't change the behaviour per test.

Reply:
How about the latest?
Its once at the function level.

Reply:
Ah, I see what you mean. Well, you can always do `module.THIS_VARIABLE = True` for testing's sake. I think that should be fine.

Reply:
Good point. Done.